core/domain/user_jobs_one_off_test.py

```diff
@@ -589,6 +598,58 @@ def test_weekly_stats_for_multiple_explorations(self):
             }
         }])
 
+    def test_stats_for_multiple_weeks(self):
```

Comment:
In response to @wxyxinyu 's [line note](https://github.com/oppia/oppia/pull/2173#discussion_r68893583):

I tried adding tests for multiple weeks, but it looks like I'm unable to simulate multiple weeks.
I've committed the code for this test so that you can take a look about what's going on.